x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/LiteralAttribute.java

```diff
@@ -14,9 +14,9 @@
 
     private final Literal literal;
 
-    public LiteralAttribute(Location location, String name, String qualifier, boolean nullable, ExpressionId id, boolean synthetic,
-            DataType dataType, Literal literal) {
-        super(location, name, dataType, qualifier, nullable, id, synthetic);
+    public LiteralAttribute(Location location, String name, String qualifier, Nullability nullability, ExpressionId id, boolean synthetic,
+                            DataType dataType, Literal literal) {
```

Comment:
Incorrect formatting?

Reply:
why? It's the method signature and the args should align.

Reply:
Because the signature is different than before; as far as I recall the alignment doesn't have to be under the same param (see the other constructors).